lib/models/barcode_plugin.dart

```diff
@@ -1,15 +1,19 @@
 import 'package:scanwedge/scanwedge.dart';
 
+// DOCS
+// https://github.com/ltrudu/ZebraEntepriseServices/blob/e35cf2ef006c7d3fd1296312da8ab91aa7b5bdba/datawedgeprofileintentswrapper/src/main/java/com/zebra/datawedgeprofileintents/SettingsPlugins/PluginScanner.java#L186
 class BarcodePlugin {
   final AimType aimType;
   final int timeoutBetweenScans;
-  final List<BarcodeLabelType>? enabledBarcodes, disabledBarcodes;
+  final List<BarcodeLabelType>? disabledBarcodes;
 
-  BarcodePlugin({this.aimType = AimType.trigger, this.timeoutBetweenScans = 0, this.disabledBarcodes, this.enabledBarcodes});
+  BarcodePlugin({this.aimType = AimType.trigger, this.timeoutBetweenScans = 0, this.disabledBarcodes});
   Map<String, dynamic> get toMap => {
         'PARAM_LIST': {
           'scanner_selection': 'auto',
           'scanner_input_enabled': 'true',
+          'inverse_1d_mode': '2',
```

Comment:
Do you know what the inverse_1d_mode and decoder_upca_peramble do? What could this break? It should probably be a property instead that can be set.